apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -100,7 +100,7 @@ int kernel_tc_main(int argc, char *argv[])
 
 #ifdef CONFIG_TC_KERNEL_LIBC_PTHREAD
 #if (!defined CONFIG_PTHREAD_MUTEX_TYPES)
-#error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC
+	/* #error CONFIG_PTHREAD_MUTEX_TYPES is needed for testing LIBC_PTHREAD TC */
```

Comment:
Yes, I agree that those lines are not valid as of now.
and with reference to `PTHREAD TC` which config you want me to check to inform the user?
If we use `CONFIG_PTHREAD_MUTEX_TYPES` check as follows, it means same as previous except the build error.
```
#ifdef CONFIG_PTHREAD_MUTEX_TYPES
 .
 .
 .
#else
             printf("tc_libc_pthread will not be executed. CONFIG_PTHREAD_MUTEX_TYPES is not set");
```
What do you say?